packages/generate-bot/main.js

```diff
@@ -23,13 +22,11 @@ const process = require("process");
 exports.checkValidity = function(testString) {
   let isValid = true;
   const relativePath = path.resolve("./packages");
-  const invalidName = /[-0-9]/;
+  const invalidName = /[0-9]/;
```

Comment:
my concern with this is that it might include `@`, `ツ`, etc., instead of `invalidName`, I would have, `validName`:

```
const validName = /[-a-z_]+/;
```

And just reverse the logic, check for validity rather than that something isn't valid.